docs/_data/2019.portland.speakers.yaml

```
@@ -701,3 +701,12 @@
     twitter: ''
     website: ''
   title: 'Product Documentation Strategy: What Is It and Why Do We Need It?'
+- abstract: "<p>As documentarians, we know that knowledge is critical: accurate, comprehensive, and  documentation drives product awareness and adoption. For engineers and other technologists, detailed documentation is essential for development velocity, code quality, and the reliability of services running in production.</p><br>
```

```suggestion
- abstract: "<p>As documentarians, we know that knowledge is critical: accurate, comprehensive, and  documentation drives product awareness and adoption. For engineers and other technologists, detailed documentation is essential for development velocity, code quality, and the reliability of services running in production.</p>
```

(this is causing excessive whitespace)

---

🎉